pkg/pillar/rootfs/usr/sbin/xl

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+# shellcheck disable=SC2046,SC2086
+exec chroot /hostfs /usr/bin/ctr --namespace services.linuxkit t exec --cwd / ${TERM:+-t} --exec-id $(basename $(mktemp)) xen-tools env ${TERM:+TERM=}$TERM $(basename $0) "$@"
```

Comment:
This seems convoluted. Any reason not to just bind-mount the `containerd.sock` file into pillar, and then do:

```
ctr --address /run/containerd/containerd.sock --namespace services.linuxkit t exec --cwd / ${TERM:+-t} --exec-id $(basename $(mktemp)) xen-tools env ${TERM:+TERM=}$TERM $(basename $0) "$@"
```

Actually, isn't `/run` already mounted in in `build.yml`? Why do we even need to `chroot` to do it?